docs/releases/v1.16.0.md

```diff
@@ -0,0 +1,64 @@
+# Networking Core Module Release 1.15.0
+
+Welcome to the latest release of the `Networking` module of [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution) maintained by team SIGHUP.
+
+This minor release updates some components and adds support to Kubernetes 1.27.
+
+## Component Images 🚢
+
+| Component         | Supported Version                                                                | Previous Version |
+| ----------------- | -------------------------------------------------------------------------------- | ---------------- |
+| `calico`          | [`v3.27.0`](https://projectcalico.docs.tigera.io/archive/v3.26/release-notes/)   | `v3.26.1`        |
+| `cilium`          | [`v1.15.0`](https://github.com/cilium/cilium/releases/tag/v1.13.1)               | `v1.13.3`        |
```

Comment:
The documentation link is pointing to an older version.
Also, the previous version is different
```suggestion
| `cilium`          | [`v1.15.0`](https://github.com/cilium/cilium/releases/tag/v1.15.0)               | `v1.14.3`        |
```

Reply:
Fixed by https://github.com/sighupio/fury-kubernetes-networking/commit/07c039cdf354b28ccd3c03623d0709a2bdb511d4